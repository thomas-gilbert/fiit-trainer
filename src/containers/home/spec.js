import React from 'react';
import { Home } from './index.js';

describe('Home Component', () => {
  it('renders correctly', () => {
    const list = {
      items: [{
        text: 'hello',
      }],
    };
    const removeExerciseFromList = () => ({});
    const component = global.shallow(
      <Home
        list={list}
        removeExerciseFromList={removeExerciseFromList}
      />
    );

    expect(component).toMatchSnapshot();
  });
});
