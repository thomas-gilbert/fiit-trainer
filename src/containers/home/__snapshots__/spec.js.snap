// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component renders correctly 1`] = `
<Row
  bsClass="row"
  componentClass="div"
>
  <Col
    bsClass="col"
    componentClass="div"
    md={12}
  >
    <div
      className="lander"
    >
      <PageHeader
        bsClass="page-header"
      >
        FIIT Trainer 
        <small>
          A programmable HIIT Trainer
        </small>
      </PageHeader>
      <Connect(Controller) />
      <List
        list={
          Object {
            "items": Array [
              Object {
                "text": "hello",
              },
            ],
          }
        }
        removeAction={[Function]}
      />
    </div>
  </Col>
</Row>
`;
